### PR TITLE
Final pass-through, G9 question amendments

### PR DESCRIPTION
**Pull request by samuelhwilliams**
* Final pass-through of content and questions for G9 applications.
* This will break existing G9 services in preview, so will require a new PR for dm-api and dm-supplier-frontend.
* Adds a security testing section to cloud-support.
* Bump version to 6.4.5.

### CODE REVIEW

File: frameworks/g-cloud-9/questions/services/approachToResilience.yml [hunk 1]
```diff
@@ -4,6 +4,11 @@ question_advice: >
   Include how your datacentre setup is resilient. If you don’t want to make this information public, you can say that
   it’s available on request.
 
+  Read about the government’s <a
+  href="https://www.ncsc.gov.uk/guidance/cloud-security-principle-2-asset-protection-and-resilience#resilience"
+  target="_blank" rel="noopener noreferrer">2nd cloud security principle: ‘Asset protection and resilience’ (link opens
+  in a new tab)</a>.
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/approachToResilience.yml]: Have we checked this works and do we really need to say "link opens in a new tab"?  If it's confusing can't we just leave it working like a normal link? :trollface:

Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/approachToResilience.yml]: And ditto for all the other places we say "link opens in a new tab"

File: frameworks/g-cloud-9/questions/services/dataProtectionBetweenNetworks.yml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 name: Data protection between the buyer’s and supplier’s networks
 question: How do you protect data between the buyer’s network and your network?
+question_advice: >
+  Read about the government’s <a
+  href="https://www.ncsc.gov.uk/guidance/cloud-security-principle-1-data-transit-protection" target="_blank"
+  rel="noopener noreferrer">1st cloud security principle: Data-in-transit protection’ (link opens in a new tab)</a>.
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/dataProtectionBetweenNetworks.yml]: Missing an opening fancy quote before `Data-in-transit...`

File: frameworks/g-cloud-9/questions/services/dataProtectionWithinNetwork.yml [hunk 3]
```diff
@@ -1,5 +1,9 @@
 name: Data protection within the supplier’s network
 question: How do you protect data within your network?
+question_advice: >
+  Read about the government’s <a
+  href="https://www.ncsc.gov.uk/guidance/cloud-security-principle-1-data-transit-protection" target="_blank"
+  rel="noopener noreferrer">1st cloud security principle: Data-in-transit protection’ (link opens in a new tab)</a>.
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/dataProtectionWithinNetwork.yml]: Missing an opening fancy quote before `Data-in-transit...`

File: frameworks/g-cloud-9/questions/services/approachToResilience.yml (hunk 1 above)
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/approachToResilience.yml]: The 'link opens in new tab' bit is best practice for accessibility according to some docs given me by @laurenceberry.

File: frameworks/g-cloud-9/questions/services/auditBuyersActions.yml [hunk 4]
```diff
@@ -1,5 +1,9 @@
 name: Access to audit information about users’ actions
 question: How do buyers access audit information about the actions their users have taken?
+question_advice: >
+  Read about the government’s <a
+  href="https://www.ncsc.gov.uk/guidance/cloud-security-principle-13-audit-information-users" target="_blank"
+  rel="noopener noreferrer">13th cloud security principle: ‘Audit information for users’ (link opens in a new tab)</a>.
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/auditBuyersActions.yml]: We're really referring to these things by number? "Ah yes, number 13, of course!"

File: frameworks/g-cloud-9/questions/services/dataProtectionBetweenNetworks.yml (hunk 2 above)
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/dataProtectionBetweenNetworks.yml]: And it's number 1 so very important to have the right quote style around it

File: frameworks/g-cloud-9/questions/services/auditBuyersActions.yml (hunk 4 above)
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/auditBuyersActions.yml]: Unlucky for some.

File: frameworks/g-cloud-9/questions/services/dataProtectionBetweenNetworks.yml (hunk 2 above)
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/dataProtectionBetweenNetworks.yml]: 👍

File: frameworks/g-cloud-9/questions/services/dataProtectionWithinNetwork.yml (hunk 3 above)
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/dataProtectionWithinNetwork.yml]: 👍

File: frameworks/g-cloud-9/questions/services/approachToResilience.yml (hunk 1 above)
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/approachToResilience.yml]: Or just not messing about with `target` in the first place :)

Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/approachToResilience.yml]: Either way; yes, tested and working.

File: frameworks/g-cloud-9/questions/services/secureDevelopment.yml [hunk 5]
```diff
@@ -1,10 +1,9 @@
 name: Approach to secure software development best practice
 question: How does your organisation demonstrate that it adheres to best practice in secure software development?
 question_advice: >
-  <a href="https://www.ncsc.gov.uk/guidance/cloud-security-standards-and-definitions" target="_blank" rel="noopener noreferrer">CESG CPA Build Standard (new tab)</a>,
-  <a href="https://www.ncsc.gov.uk/guidance/cloud-security-standards-and-definitions" target="_blank" rel="noopener noreferrer">ISO/IEC 27034 (new tab)</a>,
-  <a href="https://www.ncsc.gov.uk/guidance/cloud-security-standards-and-definitions" target="_blank" rel="noopener noreferrer">ISO/IEC 27001 (new tab)</a>, or
-  <a href="https://www.ncsc.gov.uk/guidance/cloud-security-standards-and-definitions" target="_blank" rel="noopener noreferrer">CSA CCM v3.0 (new tab)</a>
+  Read about the government’s <a
+  href="https://www.ncsc.gov.uk/guidance/cloud-security-principle-6-personnel-security" target="_blank"
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/secureDevelopment.yml]: Personnel security seems a weird link for this "secure software development" question - is it definitely the one we want?

File: frameworks/g-cloud-9/questions/services/staffSecurityClearanceChecks.yml [hunk 6]
```diff
@@ -1,5 +1,9 @@
 name: Staff security clearance checks
 question: How do you manage staff security clearance checks?
+question_advice: >
+  Read about the government’s <a
+  href="https://www.ncsc.gov.uk/guidance/cloud-security-principle-6-personnel-security" target="_blank" rel="noopener
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/staffSecurityClearanceChecks.yml]: Yeah this one seems like the right place for it.

Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/staffSecurityClearanceChecks.yml]: Good spot.

**Comment by TheDoubleK** [comment]
Think this looks good now :+1:

File: frameworks/g-cloud-9/questions/declaration/MI.yml [hunk 7]
```diff
@@ -1,5 +1,5 @@
 name: Providing management information (MI)
 question: >
   Do you confirm that you’re prepared to provide all the monthly G-Cloud&nbsp;9-related management information (MI)
-  to CCS as outlined in the <a href="/suppliers/frameworks/g-cloud-9" target="_blank" rel="noopener noreferrer">reporting template (new tab)</a>?"
+  to CCS as outlined in the <a href="/suppliers/frameworks/g-cloud-9" target="_blank" rel="noopener noreferrer">reporting template (new tab)</a>?
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/declaration/MI.yml]: I get the new tab stuff for links outside of Marketplace, but not sure we need it when the links are within our app.

Too late now for this discussion though ¯\_(ツ)_/¯

File: frameworks/g-cloud-9/questions/declaration/employersInsurance.yml [hunk 8]
```diff
@@ -1,15 +1,12 @@
...
+  members. You must answer ‘Yes’ or ‘Not applicable’ to be eligible for consideration for the G-Cloud&nbsp;9 framework.
 type: radios
 options:
-  -
-    label: "Yes – your organisation has, or will have in place, employer’s liability insurance of at least £5 million and you will provide certification prior to framework award."
-  -
-    label: "No - your organisation does not have, and will not have in place, employer’s liability insurance of at least £5 million prior to framework award."
-  -
-    label: "Not applicable - your organisation does not need employer’s liability insurance because your organisation employs only the owner or close family members."
+  - label: "Yes – your organisation has, or will have in place, employer’s liability insurance of at least £5 million and you will provide certification before the framework is awarded."
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/declaration/employersInsurance.yml]: Changing the wording here will mean the values won't pre-fill properly from earlier declarations, but fine to ask this question again for the new framework I think  ¯\\_(ツ)_/¯

File: frameworks/g-cloud-9/questions/declaration/MI.yml (hunk 7 above)
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/declaration/MI.yml]: Yes. Well pointed out though; those (new tab) bits are outdated.

**Comment by TheDoubleK** [comment]
Even better now :+1:

**Pull request merged**